api/routes/users.js

```diff
@@ -3,4 +3,33 @@ const express = require('express');
 const router = express.Router();
 const usersData = require('../data').users;
 
+/*router.post('/signup', async function(req, res){
+    let firstName = req.query.firstName;
+    let lastName = req.query.lastName;
+    let email = req.query.email;
+    
+    try{
+        let user = await usersData.addUser(firstName, lastName);
+        res.send("Successful!");
+    }catch(error){
+        res.send(error);
+    }
+    
+
+
+});*/
+
+router.get('/profile/:email', async function(req, res){
```

Comment:
not sure if passing the email in the url is good idea or not. Probably doesn't matter

Reply:
This might not be a good idea for security purposes, I've also never seen that done before, so I think its best to not reinvent the wheel. We should probably pass in a user id instead of an email.

Reply:
working on a workaround rn